From dd4a3a9f897e534be569a93f47c6f0c79a427b9f Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 02 Aug 2022 19:02:18 +0300
Subject: [PATCH] [SW-0] erom: recovery: add comment for status set

Preboot's LKD module is responsible for setting the
CPU_BOOT_STATUS_WAITING_FOR_BOOT_FIT boot flag, so LKD and hl-fw-loader
know they can start working with the FW (e.g. send boot-fit, or agent).
The eROM has no LKD module and therefore we need to set the flag
explicitly.
Add a comment about that.

Change-Id: I20ee203ddd47949e00c2aee539881722452d1c72
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ext_rom/erom_recovery.c b/app/ext_rom/erom_recovery.c
index 36e995b..4a3975a 100644
--- a/app/ext_rom/erom_recovery.c
+++ b/app/ext_rom/erom_recovery.c
@@ -119,6 +119,7 @@
 
 	preboot_clear_wd_gpio();
 
+	/* In Preboot, module LKD is responsible for setting the below flag */
 	cpu_boot_sts_set(CPU_BOOT_STATUS_WAITING_FOR_BOOT_FIT);
 
 	preboot_puts("In recovery mode\n");
